### ISSUE TITLE
fix: characteristic notifications don't work

### ISSUE THREAD

**Issue by jefflongo** [chan150/flutter_blue_plus_windows#13]
**Description**

Subscribing to a characteristic notification does not call produce a notification.

**Steps To Reproduce**

```dart
final subscription = _characteristic.onValueReceived.listen((data) => onNotification); /// !!! NOT CALLED
_characteristic.device.cancelWhenDisconnected(subscription);
try {
  await _characteristic.setNotifyValue(true); // !!! RETURNS TRUE
  print('Notifying: ${_characteristic.isNotifying}'); // !!! PRINTS FALSE
} catch (e) {
  await subscription.cancel();
  print('Failed to enable notifications: $e'); // !!! NOT CALLED
}
```

**Expected Behavior**

The expected behavior is that `onNotification` is called when the characteristic notifies. The `isNotifying` getter should return true if listing is successful. In the given sample code, `setNotifyValue` returns true, but `isNotifying` returns false. `onNotification` never gets called.

**Additional Context**

The `_isNotifying` map in `flutter_blue_plus_windows.dart` never seems to be non-empty, which is why `isNotifying` always returns false. Not sure why the `characteristicValueStreamOf` stream doesn't work though. The code I'm using works reliably on Android with the same version of FlutterBluePlus.

**Comment by chan150** [comment]
It may be resolved in `1.24.12`.
Please use newer version.

**Comment by jefflongo** [comment]
@chan150 just tried again in 1.24.13, it now reports `isNotifying` as true but I still don't get any callback on notification.
